hcmanager/validation_types.go

```diff
@@ -0,0 +1,36 @@
+package hcmanager
+
+//RAM contains ram details fetched from the introspection data
```

Comment:
If all those types are containing introspection data, could you maybe use the BMO types directly ?

Reply:
Nice Idea. I didn't think on that.

Reply:
Only question is we don't need all RAM details, currently we are only supporting Size params but looking at future proposals, good to have all those information here.

Reply:
even if you are not using every field of the type, you could still use it, it brings you closer to BMO, and avoid that when your complexity grows while supporting new fields, you diverge too much from BMO

Reply:
@maelk We removed the user defined types and used the BMO types instead. 
Kindly review commit e124097e7e8313c08b5c95ca95b1952c089a50a4